README.md

```diff
@@ -136,6 +136,8 @@ const browser = await launch();
 
 // Connect to first browser instead
 const anotherBrowser = await launch({ wsEndpoint: browser.wsEndpoint() });
+// or simply...
+const anotherBrowser2 = await launch(browser.wsEndpoint());
```

Comment:
I'm not a fan of this API change. Could we add a `endpoint` option (with `wsEndpoint` printing a "this is deprecated use `endpoint` instead") which supports both http and ws endpoints?